android-pdf-viewer/src/main/java/com/github/barteksc/pdfviewer/annotation/core/pdf/PdfUtil.kt

```diff
@@ -56,7 +58,7 @@ object PdfUtil {
 
     /** Maps shapes to annotations and draws them to the given PDF */
     @JvmStatic
-    fun drawAnnotations(pdfFile: File, outputDirectory: String, jsonShapes: String) {
+    suspend fun drawAnnotations(pdfFile: File, outputDirectory: String, jsonShapes: String) {
         AnnotationManager.removeAnnotationsFromPdf(pdfFile.path)
         val resultData = getPdfToImageResultData(pdfFile.path, outputDirectory)
         if (resultData == null) {
```

Comment:
Since this method needs to be executed in a bg thread then use a `withContext` inside here instead of forcing the caller to know about this and having an `IO` dispatcher when launching a coroutine.

The caller should not care about those details so the library is responsible to use the proper dispatcher

Reply:
Done